src/offline/cable_checks.F90

```diff
@@ -204,6 +205,8 @@ MODULE cable_checks_module
 
   TYPE(ranges_type), SAVE :: ranges
 
+  IF (cable_user%GW_MODEL) ranges%sucs = [30., 800.]   ! MMY the range [-0.8, -0.03] doesn't suit Mark Decker's version
+
```

Comment:
I like it. I would add `hyds` since it needs to be 1000 times bigger according to Mengyuan's comment.


```suggestion

  IF (cable_user%GW_MODEL) THEN
  ! The ground water model was implemented using different 
  ! units and sign convention for some parameters.
    ranges%sucs = [30., 800.]   
    ranges%hyds = ranges%hyds * 10000.
  END IF

```